com.rawtherapee.RawTherapee.yaml

```diff
@@ -0,0 +1,213 @@
+app-id: com.rawtherapee.RawTherapee
+runtime: org.gnome.Platform
+runtime-version: '3.32'
```

Comment:
Update runtime to 3.34

Reply:
Resolved in https://github.com/scx/flathub/commit/a1fec04dbc0772da22c5945d2fe159a38bde729d